developer-tools/scaleway-cli/quickstart.mdx

```diff
@@ -113,13 +149,19 @@ Refer to the [Scaleway CLI repository](https://github.com/scaleway/scaleway-cli)
 
     Press enter if the default shell type displayed is the one you are using. Otherwise, enter the correct shell type you are using.
 
-    The configuration is complete, and you are prompted with the message below:
+9. The Scaleway CLI will now check if an SSH key is present in your local machine in the `id_ecdsa.pub`, `id_ed25519.pub`, and `id_rsa.pub` files. 
+
+    If a key is present, but not associated with the default Scaleway project, the Scaleway CLI will add this key to the project defined previously.
+
+    Otherwise, the Scaleway CLI will either confirm that the key is already present in your Scaleway project, or warn you that no key were found. Refer to the [dedicated documentation](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) to create an SSH key.
```

Comment:
```suggestion
    Otherwise, the Scaleway CLI will either confirm that the key is already present in your Scaleway Project, or warn you that no key were found. Refer to the [dedicated documentation](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) to create an SSH key.
```